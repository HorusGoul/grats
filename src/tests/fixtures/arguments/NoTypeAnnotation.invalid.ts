/** @gqlType */
export default class Query {
  /** @gqlField */
  hello(args): string {
    return "Hello world!";
  }
}
