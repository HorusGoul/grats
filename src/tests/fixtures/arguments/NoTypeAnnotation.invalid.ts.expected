-----------------
INPUT
----------------- 
/** @GQLType */
export default class Query {
  /** @GQLField */
  hello(args): string {
    return "Hello world!";
  }
}

-----------------
OUTPUT
-----------------
src/tests/fixtures/arguments/NoTypeAnnotation.invalid.ts:4:9 - error: Expected GraphQL field arguments to have a TypeScript type. If there are no arguments, you can use `args: never`.

4   hello(args): string {
          ~~~~
